calendar-app/Screens/RegistrationScreen.js

```diff
@@ -26,15 +26,19 @@ const RegistrationScreen = (props) => {
   const handleLogin = async () => {
     const userData = await loginUserAPI({username});
     if(userData != null) {
-      await storeUserInfo(userData, false, true, "0");
+      if(userData.group_id !== null) {
+        await storeUserInfo(userData.id, true, true, userData.group_id);
+      }
+      else{
+        await storeUserInfo(userData.id, false, true, "0");
+      }
```

Comment:
This shouldn't be here since it is already located on main branch, try updating from main branch